### DISCUSSION TITLE
Why didn’t I get the airdrop?

### DISCUSSION THREAD

**Discussion by zhaliguo** [zkSync-Community-Hub/zksync-developers#596] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

My address
0xDa7499905ee51D3B98Af637B6215F60FfAFf4f7A

### Repo Link (Optional)

https://x.com/ThomasD99397346

**Comment by 0xsisyfos** [comment, marked as answer]
Hello,

Thank you for reaching out.

The ZK Token FAQ provides additional details on eligibility criteria: https://docs.zknation.io/zk-token/zk-token-faq

In particular, question 3 ("I met some of the eligibility criteria, but did not get an allocation, why not?") answers your question directly.

As a note, this space if for developer questions, please join the Discord for further support: https://join.zksync.dev/
